Resources/public/Admin.js

```diff
@@ -329,6 +329,11 @@ var Admin = {
 
     setup_collection_buttons: function(subject) {
 
+        // Remove previously attached event(s) which causing function to be called multiple times
+        jQuery(subject).on('click', '.sonata-collection-add', function(event){
+            return false;
+        });
+
```

Comment:
> Because the function "setup_collection_buttons" called more than once, the onclick will be triggered more than once.

@alzubaidi This is weird. Adding the same listener several times to the same element should **never** cause the handler to be called several times. That's the [DOM Events spec](https://www.w3.org/TR/DOM-Level-2-Events/events.html#Events-registration):

> If multiple identical EventListeners are registered on the same EventTarget with the same parameters the duplicate instances are discarded. They do not cause the EventListener to be called twice and since they are discarded they do not need to be removed with the removeEventListener method.